contrib/ci-builders/tekton/docker-build.sh

```diff
@@ -10,51 +35,85 @@ docker push electriccoinco/zcashd-build-debian9
 docker push electriccoinco/zcashd-build-debian11
 
 docker build . -f Dockerfile-tekton-worker --build-arg FROMBASEOS=debian --build-arg FROMBASEOS_BUILD_TAG=11 -t electriccoinco/zcashd-worker-debian11
 docker push electriccoinco/zcashd-worker-debian11
 
+docker build . -f Dockerfile-tekton-bench --build-arg FROMBASEOS=debian --build-arg FROMBASEOS_BUILD_TAG=11 -t electriccoinco/zcashd-bench-debian11
+docker push electriccoinco/zcashd-bench-debian11
+
 # Ubuntu 16.04
 docker build . -f Dockerfile-build.apt --build-arg FROMBASEOS=ubuntu --build-arg FROMBASEOS_BUILD_TAG=16.04 -t electriccoinco/zcashd-build-ubuntu1604
 docker push electriccoinco/zcashd-build-ubuntu1604
 
 docker build . -f Dockerfile-tekton-worker --build-arg BASEOS=ubuntu --build-arg FROMBASEOS=ubuntu --build-arg FROMBASEOS_BUILD_TAG=1604 -t electriccoinco/zcashd-worker-ubuntu1604
 docker push electriccoinco/zcashd-worker-ubuntu1604
 
+docker build . -f Dockerfile-tekton-bench --build-arg BASEOS=ubuntu --build-arg FROMBASEOS=ubuntu --build-arg FROMBASEOS_BUILD_TAG=1604 -t electriccoinco/zcashd-bench-ubuntu1604
+docker push electriccoinco/zcashd-bench-ubuntu1604
+
 # Ubuntu 18.04
 docker build . -f Dockerfile-build.apt --build-arg FROMBASEOS=ubuntu --build-arg FROMBASEOS_BUILD_TAG=18.04 -t electriccoinco/zcashd-build-ubuntu1804
 docker push electriccoinco/zcashd-build-ubuntu1804
 
 docker build . -f Dockerfile-tekton-worker --build-arg BASEOS=ubuntu --build-arg FROMBASEOS=ubuntu --build-arg FROMBASEOS_BUILD_TAG=1804 -t electriccoinco/zcashd-worker-ubuntu1804
 docker push electriccoinco/zcashd-worker-ubuntu1804
 
+docker build . -f Dockerfile-tekton-bench --build-arg BASEOS=ubuntu --build-arg FROMBASEOS=ubuntu --build-arg FROMBASEOS_BUILD_TAG=1804 -t electriccoinco/zcashd-bench-ubuntu1804
+docker push electriccoinco/zcashd-bench-ubuntu1804
+
 # Ubuntu 20.04
 docker build . -f Dockerfile-build.apt --build-arg FROMBASEOS=ubuntu --build-arg FROMBASEOS_BUILD_TAG=20.04 -t electriccoinco/zcashd-build-ubuntu2004
 docker push electriccoinco/zcashd-build-ubuntu2004
 
 docker build . -f Dockerfile-tekton-worker --build-arg BASEOS=ubuntu --build-arg FROMBASEOS=ubuntu --build-arg FROMBASEOS_BUILD_TAG=2004 -t electriccoinco/zcashd-worker-ubuntu2004
 docker push electriccoinco/zcashd-worker-ubuntu2004
 
+docker build . -f Dockerfile-tekton-bench --build-arg BASEOS=ubuntu --build-arg FROMBASEOS=ubuntu --build-arg FROMBASEOS_BUILD_TAG=2004 -t electriccoinco/zcashd-bench-ubuntu2004
+docker push electriccoinco/zcashd-bench-ubuntu2004
+
+# Ubuntu 22.04
+docker build . -f Dockerfile-build.apt --build-arg FROMBASEOS=ubuntu --build-arg FROMBASEOS_BUILD_TAG=22.04 -t electriccoinco/zcashd-build-ubuntu2204
+docker push electriccoinco/zcashd-build-ubuntu2204
+
+docker build . -f Dockerfile-tekton-worker --build-arg BASEOS=ubuntu --build-arg FROMBASEOS=ubuntu --build-arg FROMBASEOS_BUILD_TAG=2204 -t electriccoinco/zcashd-worker-ubuntu2004
```

Comment:
Why do the build and worker/bench lines have different `FROMBASEOS_BUILD_TAG` values (both for these new ones, and for all the existing ones)? The build variant has an additional `.` present.

Reply:
I added in the missing docker files that some how got nuked on merging some branches. They are used to layer the builds per update but newer conventions might not need these injected as ARG.

The . at the end of the docker build command tells Docker that it should look for the Dockerfile in the current directory. IIRC, this is also an older convention.

Reply:
What I meant is that one of the lines uses `22.04` and the others use `2204`, for the same configuration option (which from its name sounds like a git tag, so I'd expect only one of the two variants to be valid).